shared/actions/favorite.js

```diff
@@ -79,6 +79,7 @@ const folderToProps = (folders: Array<Folder>, username: string = ''): FolderPro
     showingPrivate: true,
     smallMode: false,
     showComingSoon: !flags.tabFoldersEnabled,
+    username,
```

Comment:
Username was added to folder props so that the help screen could show it in example commands. This relationship between the store and the folder props needs to be broken eventually.